pyproject.toml

```diff
@@ -48,7 +48,7 @@ classifiers = [
 [tool.poetry.dependencies]
 python = ">=3.8, <3.11"
 
-lava-nc = { git = "https://github.com/lava-nc/lava.git", branch = "main", develop = true }
+lava-nc = "0.7.0"
```

Comment:
Can you elaborate on why should we point to the previous release? I think we should keep the dependency to main until the next release.

Reply:
Oops, didn't mean to check that in. Fixed.